src/qt/locale/bitcoin_nl.ts

```diff
@@ -1530,103 +1530,103 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Public Key:</source>
-        <translation type="unfinished"></translation>
+        <translation>Openbare sleutel:</translation>
     </message>
     <message>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Sluit</translation>
```

Comment:
Sluiten

Reply:
`Sluiten` translates into Closing, so would be better to keep it as `Sluit`